lib/ui/views/profile/user_favourites_view.dart

```diff
@@ -47,6 +48,27 @@ class _UserFavouritesViewState extends State<UserFavouritesView>
           });
         }
 
+        if (_items.isEmpty) {
+          return Center(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Text(
+                  'No Favourites',
+                  style: TextStyle(fontSize: 25, color: Colors.white),
```

Comment:
I think using `CVtheme` is better. Also since you have explicitly used `Colors.white` doesn't it remain the same even in light mode? and white on a white background makes it hard to read.

Reply:
ahhhh, my bad :(. Will make the changes